src/bandersnatch/main.py

```diff
@@ -108,6 +108,11 @@ def main():
         default=False,
         help="Do not download or delete files",
     )
+    v.add_argument(
+        "--proxy",
+        type=str,
+        help="Specify proxy server if required",
```

Comment:
Can we please give an example to make it easier for people to use. Proxy server URLs differ everywhere.

Reply:
Hello,
Certainly; I'm happy to do whatever is required! Regarding the PR, I'm a bit new to these. I originally did one for the first one (#59) but my subsequent commits have all been automatically absorbed onto the PR. Do I need to create a specific tag for each PR?

Once I know what I should be doing, I can cancel this PR and get separate PRs for each individual issue done properly. Also, I'll provide usage for proxy as requested.

Cheers!